static/templates/actors/ability-builder.html

```diff
@@ -135,7 +135,7 @@ <h2>{{localize "PF2E.Actor.Character.AbilityBuilder.Flaws"}}</h2>
                 <div class="full-row">{{localize "PF2E.Actor.Character.AbilityBuilder.BackgroundMissingHelp"}}</div>
             {{/if}}
 
-            <div class="hover-content" class="background-tooltip">
+            <div class="hover-content" id="background-tooltip">
```

Comment:
The value of id must be unique within the entire page. This shouldn't use it.

Reply:
That said, it seems that data-tooltip-content seems to use ids everywhere, even in the parts that should be unique. I guess without using tooltipster document side, it has to use ids unfortunately.

Might be better to use .tooltipster() for this like most of the codebase uses?

Reply:
It is using tooltipster like most of the rest of the codebase uses.

Reply:
I meant feeding in the content directly via the ability builder application. But making the ids unique is fine enough.